src/Components/AddItem.jsx

```diff
@@ -1,5 +1,76 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import firebase from 'firebase';
+import { db } from '../lib/firebase';
 
 export default function AddItem() {
-  return <div>Hi from Add an Item View!</div>;
+  const [token, setToken] = useState('test-token');
+  const { register, handleSubmit, watch, errors } = useForm();
+  const onSubmit = (data) => {
+    const currentList = db.collection('lists').doc(token);
+    currentList
+      .update({
+        items: firebase.firestore.FieldValue.arrayUnion({
+          name: data.itemName,
+          frequency: parseInt(data.itemFrequency),
+          lastPurchased: null,
+        }),
+      })
+      .then(() => alert('Item has been submitted!'));
+  };
+
+  return (
+    <form onSubmit={handleSubmit(onSubmit)}>
```

Comment:
This form looks great! I haven't used React Hooks Form before, it's super neat! This is more of an extra credit (aka not a blocker), but it'd be cool to leverage some of the accessibility features they have to make our app more usable for everyone. Here's the [link](https://react-hook-form.com/advanced-usage/#AccessibilityA11y) to the official React Hooks Form docs about accessibility.

Reply:
This is a great idea! We have altered the error validations to be more accessible.